storage/server/backend.go

```diff
@@ -233,7 +233,7 @@ func (s *server) AioControllerCreate(ctx context.Context, in *pb.AioControllerCr
 	params := BdevAioCreateParams{
 		Name:      in.GetDevice().GetName(),
 		BlockSize: 512,
-		Filename:  "/tmp/aio_bdev_file",
+		Filename:  in.GetDevice().GetFilename(),
```

Comment:
there is another location with `/tmp/aio_bdev_file` below

Reply:
👍 fixed